### PR TITLE
JSON config + MIME types + Logger + Properties

### PR DESCRIPTION
**Pull request by gkhom**
This PR implements: 
- JSON config for digital objects
- MIME type support (from JSON config)
- logging
- properties (app config)

### CODE REVIEW

File: test-resources/staged-test1/do1/cfg.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "namespace" : "easy-dataset",
+  "files" : [
```
Comment (janvanmansum) [inline test-resources/staged-test1/do1/cfg.json]: I'd call this datastreams, to conform to Fedora terminology.

Comment (gkhom) [inline test-resources/staged-test1/do1/cfg.json]: Changed `files` to `datastreams` and `filename` to `name`.

File: application.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+fedora-host=http://deasy:8080/fedora
```
Comment (janvanmansum) [inline application.properties:1]: fedora-url would be more correct. The value is more than only a host name.

File: pom.xml [hunk 3]
```diff
@@ -24,6 +24,21 @@
...
+            <artifactId>json4s-native_2.11</artifactId>
+            <version>3.2.11</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.1.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
```
Comment (janvanmansum) [inline pom.xml:38]: Why do we need this? In other words: what lib uses JCL logging?

**Pull request merged**